### ISSUE TITLE
Indicators not visible

### ISSUE THREAD

**Issue by JoseExposito** [elementary/greeter#582]
### What Happened?

Indicators (network, Bluetooth, accessibility, etc) are not visible after switching on the computer. 

### Steps to Reproduce

Just turn on the computer. The indicators are not visible until a Bluetooth device is connected.

### Expected Behavior

The indicators should be always visible.

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

**Comment by vjr** [comment]
I've noticed a delay (maybe 30 sec) in the wingpanel showing up on the login screen too, likely with latest updates - but i dont use bt, i've kept it turned off always. wonder what is the issue. this is on my desktop pc which does have a wifi+bt PCIe card. will also check my laptop and see if i can post any findings later on. im running odin daily on both devices.

**Comment by vjr** [comment]
OK here are some lines from `/var/log/syslog` which I hope are relevant...

```
Nov 12 13:07:56 firebird systemd[1]: systemd-hostnamed.service: Succeeded.
Nov 12 13:08:16 firebird xdg-desktop-por[943]: Failed to create settings proxy: Error calling StartServiceByName for org.freedesktop.impl.portal.desktop.gtk: Timeout was reached
Nov 12 13:08:16 firebird dbus-daemon[874]: [session uid=114 pid=874] Activating via systemd: service name='org.freedesktop.impl.portal.desktop.elementary.settings-daemon' unit='io.elementary
.settings-daemon.xdg-desktop-portal.service' requested by ':1.26' (uid=114 pid=943 comm="/usr/libexec/xdg-desktop-portal " label="unconfined")
Nov 12 13:08:16 firebird systemd[854]: Starting Portal service (Pantheon Settings Daemon)...
Nov 12 13:08:16 firebird dbus-daemon[874]: [session uid=114 pid=874] Successfully activated service 'org.freedesktop.impl.portal.desktop.elementary.settings-daemon'
Nov 12 13:08:16 firebird systemd[854]: Started Portal service (Pantheon Settings Daemon).
Nov 12 13:08:41 firebird xdg-desktop-por[943]: Failed to create file chooser proxy: Error calling StartServiceByName for org.freedesktop.impl.portal.desktop.gtk: Timeout was reached
Nov 12 13:08:41 firebird xdg-desktop-por[943]: No skeleton to export
Nov 12 13:08:56 firebird systemd[854]: xdg-desktop-portal.service: start operation timed out. Terminating.
Nov 12 13:08:56 firebird systemd[854]: xdg-desktop-portal.service: Failed with result 'timeout'.
Nov 12 13:08:56 firebird systemd[854]: Failed to start Portal service.
Nov 12 13:09:08 firebird systemd[1]: Stopping Session c1 of user lightdm.
Nov 12 13:09:08 firebird rtkit-daemon[869]: Supervising 4 threads of 1 processes of 1 users.
Nov 12 13:09:08 firebird systemd[1]: Created slice User Slice of UID 1000.
```

I started my laptop just before 13.08 and just waited for wingpanel to appear which it did around 20-25 seconds later around 13.08.16 where the settings proxy failure is seen.

I then continued to wait until the clock went just past 13.09 to login (to try and minimise other log messages) so the logs from 13.09.08 onward are from when I logged in.

Note that while waiting at the login screen after wingpanel appeared, there is another failure log about file chooser proxy around 13.08.41 timestamp... so another maybe 30 seconds timeout somewhere?

Hope this helps.

**Comment by anzyelay** [comment]
i got this problem when i upgrade the libgranite to 6.1.2  ,it is okey to compile wingpanel by using granite(v<6.1.2)  ,or  try not use  granite( getting dark theme style ) in wingpanel making a difference.that what i meet

**Comment by 4jNsY6fCVqZv** [comment]
> I've noticed a delay (maybe 30 sec) in the wingpanel showing up on the login screen too, likely with latest updates - but i dont use bt, i've kept it turned off always. wonder what is the issue. this is on my desktop pc which does have a wifi+bt PCIe card. will also check my laptop and see if i can post any findings later on. im running odin daily on both devices.

The length of the delay is approximately the same for me. And Bluetooth is deactivated by default for me as well.
I use the stable version of Odin including the latest updates.

**Comment by vjr** [comment]
For the rest of us the issue is likely fixed by https://github.com/elementary/wingpanel/pull/428 but the reporter @JoseExposito mentioned the indicators show up only after connecting a bluetooth device so this issue might still be unresolved, unless Jose says otherwise.

**Comment by JoseExposito** [comment]
It happens after a timeout also on my side. The Bluetooth connection was just a coincidence.

**Comment by lenemter** [comment]
Closing because I cannot reproduce the issue in OS 8. If you still encounter the issue, please reply here
